jscomp/core/js_dump_program.ml

```diff
@@ -73,8 +73,12 @@ let node_program ~output_dir f (x : J.deps_program) =
   P.newline f;
   let cxt =
     Js_dump_import_export.requires L.require Ext_pp_scope.empty f
-      (Ext_list.map x.modules (fun x ->
-           ( x.id,
+      (* dynamic import should not print in require statement *)
```

Comment:
not be emitted in require statements

Reply:
Oh actually, it happened. Not only es6_program.

Reply:
Sorry I just meant to suggest an alternative phrasing of the comment

Reply:
Got it. https://github.com/rescript-lang/rescript-compiler/pull/6232/commits/36038524bb53874a2299e3993ec6b4cc5e244965